src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -127,7 +127,7 @@ public void ValidPathWithTrailingSlash()
         }
 
         [ConditionalFact(nameof(UsingNewNormalization))]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
```

Comment:
I'm not sure I'm just doing the clean up to only have `TargetFrameworkMonikers.Uap` as UapAot flag is included on Uap's flag now. @JeremyKuhne are this tests skipped because of long path support ([you marked this tests as skip here](https://github.com/dotnet/corefx/commit/551fe49174378adcbf785c0ab12fc69355cef6e8#diff-d05900b0bb0438606e997c81e59e05a4R233)) or this is because of AppContainer?